.github/workflows/validate.yml

```diff
@@ -1,29 +0,0 @@
-name: Validate
-
-on:
-  push:
-    branches:
-      - main
-  pull_request: {}
```

Comment:
I think we should keep this? This is the part which tells GitHub Actions to run on each PR.